packages/boxel/addon/components/boxel/field/index.css

```diff
@@ -1,4 +1,6 @@
 .boxel-field {
+  --boxel-field-label-align: top;
```

Comment:
`top` isn’t supported for `align-items`, should it be [`normal`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#formal_definition), which is the default?

Reply:
d'oh, good catch